src/table/toolbar/table-toolbar.component.ts

```diff
@@ -74,6 +79,10 @@ export class TableToolbar {
 	@Input() set cancelText(value: { CANCEL: string }) {
 		this._cancelText.override(value.CANCEL);
 	}
+	/**
+	 * Will be emitted when cancel button is clicked and all rows are deselected.
+	 */
+	@Output() onCancelSelected = new EventEmitter();
```

Comment:
we probably want to remove this if we're running everything through the model ... this doesn't seem to be hooked up to anything anyway